packages/website/src/examples/layout/ModalWizard/ModalWizardWithReactHookForm.demo.tsx

```diff
@@ -0,0 +1,104 @@
+import {Button, Flex, ModalWizard, ModalWizardStepProps, TextInput, Title} from '@coveord/plasma-mantine';
+import {useState} from 'react';
+import {Controller, useForm} from 'react-hook-form';
```

Comment:
I think @GermainBergeron's comment is good.

You should replace ReactHookForm by MantineForm for your example, it would make it easier.
At the moment ReactHookForm is not our official method.

Reply:
I see. It's been corrected 👍